README.md

```diff
@@ -21,6 +21,12 @@ What you need to have installed:
 * zlib
 * opus (libopus-dev)
 
+Debian/Ubuntu:
+```console
+sudo apt-get install qtbase5-private-dev zlib1g-dev libopenal-dev libavcodec-dev libavresample-dev libswscale-dev libopenal-data libopenal1 libavutil-dev
+```
```

Comment:
Nah, it isn't a proper usage of `console`. Compare these:

```console
sudo apt-get install ...
```

and this:

```console
# apt-get install ...
```

It gets the correct coloring if starting every line with `$` or `#`.